src/python/zquantum/core/bitstring_distribution/distance_measures/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .clipped_negative_log_likelihood import *
-from .mmd import *
+from .clipped_negative_log_likelihood import compute_clipped_negative_log_likelihood
+from .mmd import compute_mmd, compute_multi_rbf_kernel, compute_rbf_kernel
```

Comment:
Good catch! It's not always feasible to replace the wildcard with explicit imports, but here it totally makes sense.